### DISCUSSION TITLE
All network connections close down including VNC

### DISCUSSION THREAD

**Discussion by FStars** [transmission/transmission#8507] [Support]
### What is the issue?

This problem has existed since 4.0.5. 

I am running Transmission on a Mac Mini server, M2 Pro, now MacOS 14.6.1. 

The problem started 4 months ago. Transmission will run 1-2 or max 3 days, then the server will close all network connections including VNC. Samba will run.

The server is running Plex (latest version) and PIA (latest version) with split tunnel mode turned on and  kill switch turned off. 

I have tried to run without Plex and it still happens. 

I have reinstalled MacOS, and it still happens. 

I have sent log files to Apple and they tell me that an application is the cause of the problem and not MacOS.  

I have sent debug log files to PIA and they tell me, that they can't even see, that the connection is closed. I don't know why but my PIA app still have "a green light" turned on when it happens, meaning that the application has not detected, that the connection is down. 

Using a browser gives me a "no network connection".

I can't ping an ip neither on wan nor lan

The only solution is to use qBittorrent. Then everything is stable - and I'm right now into my third week of stable operation with qBittorrent meaning I have not quit qBittorrent. It just runs and so do the network. By stable I mean, that I have not quit the application at all. If I startup Transmission it will run on the server for a maximum of 3 days. Sometimes only an hour before all network connections close down. 

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.6

**Comment by killemov** [comment]
> The problem started 4 months ago. Transmission will run 1-2 or max 3 days, then the server will close all network connections including VNC. Samba will run.

That DOES smell like a server issue. Can you trace back if you did any updates 4 months ago? (What changed? New router or other network related changes?) Can you make more service connections (telnet, nfs, monitoring tools, ...) and see if they also break at the same time?

**Comment by FStars** [comment]
I agree. But Apple Support says, that it is caused by an application. If I startup Transmission it WILL happen in 1-2-3 days. That's 100 percent sure. If I don't startup Transmission the server will run with no problems for days. Right now qBittorrent has been running continuous since August 2nd - almost 4 weeks. I have downloded several files and I am seeding 73 files with no problems at all. I am sure that Transmission is causing the network error. The only two other apps running are PIA and Plex.

**Comment by FStars** [comment]
Telnet breaks. Network monitoring tool breaks. I have tried several times to reboot my Netgear Orbi 850 router right after the incident happens but that did not help. All other Macs on the same network runs with no problems when the incident happens and they are alle connected to the internet. That's why I am focussing on the server and what goes on there.
